### PR TITLE
EREGCSC-1838 Enable text within windows/appendix

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC-1838

**Description-**
If I'm searching for some keywords that happen to be within a table or another form of special text within the regulations, I want to see that information in my search results.

**This pull request changes...**

- Search results will look at tables for full text search in regulations
- Search results will look at appendix for full text search in regulations
- Search results will look at formulas for full text search in regulations

**Steps to manually verify this change...**

1. steps to view and verify change
Make the following searches, and this is where they should appear

Tables - "Individuals with family income 101–150% of the FPL" should bring up 447.52, but doesn't
Formulas - "state per capita income" should bring up 433.10, but doesn't
Lists - "Eligibility of deemed beneficiaries of AFDC" should bring up 436.2, but doesn't
Appendix - "time not to be sterilized" should bring up Appendix to Subpart F of Part 441, but doesn't
Effective date note - "contains information collection" should bring up 441.351, but doesn't
Footnote - "regulations formerly contained in" should bring up 432.10, but doesn't

### CODE REVIEW

File: solution/backend/regcore/search/migrations/0008_alter_searchquery.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.18 on 2023-04-12 15:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```
Comment (thwalker6) [inline solution/backend/regcore/search/migrations/0008_alter_searchquery.py:8]: The search vector field is a generated column.  In order to alter a column related to it you need to drop the column and then repopulate it.  Legnth had to be modified for section string so the column is just dropped, field altered, then readded with existing calculations.

File: solution/ui/regulations/eregs-vite/src/components/search/RegResults.vue [hunk 2]
```diff
@@ -22,21 +22,29 @@ const partDocumentTitleLabel = (string) => string.toLowerCase();
...
 const createResultLink = (
-    { headline, title, part_number, section_number, date },
+    { headline, title, part_number, section_number, date, section_title },
     base,
     query
 ) => {
     // get highlight content from headline
     const highlightedTermsArray = getTagContent(headline, "search-highlight");
-
+    const q = query.replace("%", "%25")
```
Comment (PhilR8) [inline solution/ui/regulations/eregs-vite/src/components/search/RegResults.vue:31]: I'd prefer something more descriptive that `q` here for a variable name

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/js/naming-conventions-js.html

File: solution/ui/regulations/eregs-vite/src/components/search/RegResults.vue [hunk 3]
```diff
@@ -22,21 +22,29 @@ const partDocumentTitleLabel = (string) => string.toLowerCase();
...
 const createResultLink = (
-    { headline, title, part_number, section_number, date },
+    { headline, title, part_number, section_number, date, section_title },
     base,
     query
 ) => {
     // get highlight content from headline
     const highlightedTermsArray = getTagContent(headline, "search-highlight");
-
+    const rawQuery = query.replace("%", "%25")
```
Comment (PhilR8) [inline solution/ui/regulations/eregs-vite/src/components/search/RegResults.vue:31]: love it!

**Review by PhilR8 — APPROVED** [review]
We went over this PR yesterday at pairing and there were no complaints, and my one minor comment was addressed.  LGTM!

**Pull request merged**